CWE-94-Python_darwin/cloudmesh-deprecated_cloudmesh-iu/j.py

```
@@ -0,0 +1,78 @@
+# command = f'ssh juliet "ssh {host} ./ENV3/bin/jupyter-lab --ip 0.0.0.0 --port {port}"'
+
+# command = f'jupyter-lab --ip localhost --port {port} --no-browser'
+
+
+localcommand = "ssh -L 9000:{host}:9000 -i {file} juliet"  # juliet = <username>@juliet.futuresystems.org
+
+
+def live(command):
+    file = None
+    localhost = None
+    process = subprocess.Popen(shlex.split(command),
```

Ruff (0.13.2)

`subprocess` call: check for execution of untrusted input

(S603)